src/main/app/src/static/smart-documents-result.html

```diff
@@ -0,0 +1,141 @@
+      }
+
+      .box {
+        background-color: #212121;
+        font-size: 14px;
+        color: white;
+        padding: 14px 16px;
+        margin-bottom: 16px;
+      }
+
+      .mdl-layout-title__zac {
+        font-weight: 500;
+      }
+    </style>
+    <script>
+      document.addEventListener("DOMContentLoaded", function () {
+        const urlParams = new URLSearchParams(window.location.search);
+        const nameValue = urlParams.get("zaak") || urlParams.get("taak");
+        const typeValue = urlParams.get("zaak") ? "zaak" : "taak";
+        const docValue = urlParams.get("doc");
+        const resultValue = urlParams.get("result");
+        const validResults = ["success", "failed", "canceled"];
+
+        const resultText = {
+          success: {
+            en: `Document "${docValue}" has been successfully added to ${typeValue} "${nameValue}".`,
+            nl: `Document "${docValue}" is succesvol toegevoegd aan ${typeValue} "${nameValue}".`,
+          },
+          failed: {
+            en: `Failed to add document "${docValue}" to ${typeValue} "${nameValue}".`,
+            nl: `Toevoegen van document "${docValue}" aan ${typeValue} "${nameValue}" is mislukt.`,
+          },
+          canceled: {
+            en: `Adding document "${docValue}" to ${typeValue} "${nameValue}" has been canceled.`,
+            nl: `Toevoegen van document "${docValue}" aan ${typeValue} "${nameValue}" is afgebroken.`,
+          },
+          onbekend: {
+            en: `Unknown error occurred.`,
+            nl: `Onbekende fout opgetreden.`,
+          },
+        };
+
+        closingText = {
+          en: `Close this tab to resume where you left off.`,
+          nl: `Sluit dit tabblad om verder te gaan waar je bent gebleven.`,
+        };
+
+        const language = ["en", "nl"].includes(navigator.language.split("-")[0])
+          ? navigator.language.split("-")[0]
+          : "nl";
+
+        const resultMessage = validResults.includes(resultValue)
+          ? resultText[resultValue][language]
+          : resultText.onbekend[language];
+
+        document.getElementById("box").textContent = resultMessage;
+        document.getElementById("closing-text").textContent =
+          closingText[language];
+        document.title = `${typeValue.charAt(0).toUpperCase() + typeValue.slice(1)} | ${nameValue}`;
+        document.getElementById("layout-title").textContent = document.title;
+      });
+    </script>
+  </head>
+  <body>
+    <div class="mdl-layout mdl-js-layout mdl-layout--fixed-header">
+      <header class="mdl-layout__header">
+        <div class="mdl-layout__header-row">
+          <span
+            id="layout-title"
+            class="mdl-layout-title mdl-layout-title__zac"
+          >
+          </span>
+        </div>
+      </header>
+
+      <main class="mdl-layout__content">
+        <div class="page-content">
+          <div class="wrapper">
+            <div id="box" class="box">Wachten op resultaat...</div>
```

Comment:
Not language dependent "waiting for result" message?

Reply:
Should not be visible actually; default loading text; but Fixed anyways